src/app/public/modules/action-button/action-button-container.component.ts

```diff
@@ -90,7 +95,10 @@ export class SkyActionButtonContainerComponent implements AfterContentInit, OnIn
     private actionButtonAdapterService: SkyActionButtonAdapterService,
     private changeRef: ChangeDetectorRef,
     private coreAdapterService: SkyCoreAdapterService,
-    private themeSvc: SkyThemeService
+    private hostElRef: ElementRef,
+    private themeSvc: SkyThemeService,
```

Comment:
Mind putting the newer injections after the original ones (`ElementRef` appears before `SkyThemeService`) in case these injections are getting mocked out elsewhere?